### ISSUE TITLE
Yacatet's bank does nothing.

### ISSUE THREAD

**Issue by technolem** [Ruin0x11/ElonaPlusCustom_GX#10]
Using yacatect's bank just exits the inventory menu, instead of popping up the dialogue to input/withdraw cash.

**Comment by Mirarara** [comment]
It's probably due to in action.hsp line 11349
```
		if ( stat == 0 ) {
			gosub *screen_draw
			goto *pc_turn
		}
		gosub *prompt_key
```
the prompt_key and the pc_turn is reversed?
It should be
```
		gosub *prompt_key
		if ( stat == 0 ) {
			gosub *screen_draw
			goto *pc_turn
		}
```

**Comment by Ruin0x11** [comment]
Closed by #13.
